db/models/topics.models.js

```diff
@@ -85,3 +85,31 @@ exports.retrieveAllArticles = () => {
       next(err);
     });
 };
+
+exports.retrieveCommentsById = (article_id) => {
+  return db.query(`SELECT comment_id,votes,created_at,author,body,article_id FROM comments WHERE article_id = $1 ORDER BY created_at DESC`,[article_id])
+  .then((comments)=>{
+    return comments.rows
+  })
+}
+
+exports.retrieveCommentsById = (article_id) => {
+  let queryString = "SELECT comment_id,votes,created_at,author,body,article_id FROM comments";
+  const queryValues = [];
+  const queryPromises = [];
+  if (article_id) {
```

Comment:
Surely there's always going to be an article_id in some form? Do you need this if statement?